public/apps/configuration/panels/auth-view/auth-view.tsx

```diff
@@ -40,17 +43,24 @@ export function AuthView(props: AppDependencies) {
 
         setAuthentication(config.authc);
         setAuthorization(config.authz);
+        setErrorFlag(false);
+        setAccessErrorFlag(false);
       } catch (e) {
         console.log(e);
+        // requests with existing credentials but insufficient permissions result in 403, remote data-source requests with non-existing credentials result in 400
+        if (e.response && [400, 403].includes(e.response.status)) {
+          setAccessErrorFlag(true);
+        }
+        setErrorFlag(true);
```

Comment:
Should this be else? Or doesn't matter too much right?

Reply:
doesn't matter

Reply:
it is a generic error flag that should be set regardless of what type of error is caught. I've added an error flag specific to access errors to handle the display of the message.